src/seedu/addressbook/commands/Street.java

```diff
@@ -0,0 +1,16 @@
+package seedu.addressbook.data.person;
+ 
+ public class Street {
+ 	private final String street;
+ 
+ 	public Street(String street) {
+ 		this.street = street;
```

Comment:
indentation is off here! setup eclipse to use 4 space indentation rather than tabs; do it now :-P